routes/user.js

```diff
@@ -1,6 +1,36 @@
 const router = require("express").Router();
 const { User, validate } = require("../models/user");
 const bcrypt = require("bcrypt");
+const nodemailer = require("nodemailer");
+
+const otp = Math.floor(1000 + Math.random() * 9000); // generate a 4-digit OTP
+
+const sendEmail = (user) => {
+  return new Promise((resolve, reject) => {
+    const transporter = nodemailer.createTransport({
+      service: "gmail",
+      auth: {
+        user: process.env.myEmail,
+        pass: process.env.myPassword,
```

Comment:
Please use the standard format of the env config on how it was written (e.g. JWTPRIVATEKEY, JWTPRIVATEKEY)